ug-big-data/charter.md

```diff
@@ -0,0 +1,56 @@
+# Big Data User Group Charter
```

Comment:
I would like to follow the pattern of working groups, and not include a charter. Just a README. Some of the non-boilerplate could be moved into a purpose section of the README.

Example WG readme:
https://github.com/kubernetes/community/tree/master/wg-iot-edge

Definitely we shouldn't be linking to SIG governance docs, and the UG shouldn't have subprojects in kubernetes github orgs.